config/locales/it.yml

```diff
@@ -106,6 +137,7 @@ it:
         name: Nome
       spree/return_authorization:
         amount: Quantità
+        pre_tax_total: Totale al lordo delle tasse
```

Comment:
"Totale al lordo delle tasse" actually means "Total with tax included" which seems to me a little bit conflicting with the key name `pre_tax_total`